src/test/groovy/com/gooddata/visualizationObject/VisualizationClassTest.groovy

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.visualizationObject;
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.apache.commons.io.FileUtils
+import spock.lang.Specification
+
+class VisualizationClassTest extends Specification {
+    private static final String TABLE_VISUALIZATION_CLASS = "/visualizationObject/tableVisualizationClass.json"
+    private static final String BAR_VISUALIZATION_CLASS = "/visualizationObject/barVisualizationClass.json"
+    private static final String EXTERNAL_VISUALIZATION_CLASS = "/visualizationObject/externalVisualizationClass.json"
+    ObjectMapper MAPPER = new ObjectMapper()
+
+
+    private VisualizationClass visualizationClass
+
+
+    def "should serialize full"() {
```

Comment:
pls look at i.e. `AfmTest` for inspiration and use `jsonEquals` and `readObjectFromResource`

Reply:
👍